### ISSUE TITLE
Sorting clients by IP-address is incorrect

### ISSUE THREAD

**Issue by UnitedMarsupials** [sensu/uchiwa#527]
When I ask Uchiwa to sort the clients by IP-address, it performs a literal sort, resulting in 172.1.1.100 to be listed above 172.1.1.2, for example.

The dictionary sorting may be appropriate for other columns, but IP-addresses should be sorted smarter.

(Actually, when sorting by hostname, it should, probably, group hosts in the same domain together -- sorted within each domain. But our installation has only one domain, so I don't care...)

**Comment by mhelmetag** [comment]
Would it be important to have ipv6 also sort correctly? I'm not exactly sure what a table of ipv6 addresses would look like unsorted and sorted.

I have a little mockup in angular for a custom ipv4 comparator and it seems to work so I think I could get that into `uchiwa-web`.

**Comment by UnitedMarsupials** [comment]
> Would it be important to have ipv6 also sort correctly?

Yes, for completeness the application should be handling IPv6 addresses correctly too. Personally, however, I do not really care, because we aren't planning to start using IPv6 in the foreseeable future.

> I think I could get that into uchiwa-web.

Awesome!
